vnext/include/ReactUWP/XamlView.h

```diff
@@ -30,4 +31,22 @@ inline void SetTag(XamlView view, winrt::IInspectable tag)
   SetTag(view, tag.as<winrt::IPropertyValue>().GetInt64());
 }
 
+inline bool IsValidTag(winrt::IPropertyValue value)
+{
+  assert(value);
+  return (value.Type() == winrt::PropertyType::Int64);
+}
+
+inline int64_t GetTag(winrt::IPropertyValue value)
+{
+  assert(value);
+  return value.GetInt64();
+}
+
+inline winrt::IPropertyValue GetPropertyValue(winrt::FrameworkElement fe)
+{
+  assert(fe);
+  return fe.GetValue(winrt::FrameworkElement::TagProperty()).try_as<winrt::IPropertyValue>();
+}
```

Comment:
What was the scenario where we need a new GetTag/GetPropertyValue rather than using the existing GetTag ?   I was confused why we have new global functions for this that are used from one new helper location which after using GetPropertyValue immediately calls GetTag.

Reply:
GetTag assume tag is int64, but actual tag can be anything. If customer has a existing control and export it by native module, it may crash

---
In reply to: [295389047](https://github.com/microsoft/react-native-windows/pull/2623#discussion_r295389047) [](ancestors = 295389047)